theano/sandbox/cuda/tests/test_cusolver.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, division, print_function
+
+import unittest
+import numpy
+import theano
+
+from theano.tests import unittest_tools as utt
+
+# Skip tests if cuda_ndarray is not available.
+from nose.plugins.skip import SkipTest
+import theano.sandbox.cuda as cuda_ndarray
+from theano.misc.pycuda_init import pycuda_available
```

Comment:
You can remove that pycuda import and tests. The cusolber_available will do the work.